src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Person } from '../types/Person';
+
+type Props = {
+  searchPeople: string;
+  handleSearchChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  handleUsePeople: () => void;
+  usePeople: boolean;
+  peopleFilter: Person[];
+  handleClickUser: (person: Person) => void;
```

Comment:
Do you need many props?
Try to simplify your code. You need to pass as props only people, onSelected function and delay